xml/net/server/protocol.xml

```diff
@@ -29,26 +32,31 @@
     </enum>
 
     <enum name="AvatarChangeType" type="char">
+        <comment>How a player's appearance is changing</comment>
         <value name="Clothes">1</value>
         <value name="Hair">2</value>
         <value name="HairColor">3</value>
     </enum>
 
     <enum name="TalkReply" type="short">
+        <comment>Reply code sent with TALK_REPLY packet</comment>
         <value name="NotFound">1</value>
     </enum>
 
     <enum name="SitState" type="char">
+        <comment>Indicates how a player is sitting (or not sitting)</comment>
         <value name="Stand">0</value>
         <value name="Chair">1</value>
         <value name="Floor">2</value>
     </enum>
 
     <enum name="MapEffect" type="char">
+        <comment>An effect that occurs on a map</comment>
```

Comment:
Might want to specify "for all players on a map" to say it's for the whole map rather than local